### ISSUE TITLE
request: bioconda

### ISSUE THREAD

**Issue by snewhouse** [Yandell-Lab/VVP-pub#3]
Hi
Great tool - any chance you can or will get this up as a package in bioconda?
Would be great for the bioinformatics and NGS community

Best

Steve

**Comment by stevendflygare** [comment]
I agree, getting this into bioconda would be a great community resource.  We don't have any current development plans with this in mind, but we'll put in on the list.
